src/appleseed.studio/python/pythoninterpreter.h

```diff
@@ -32,6 +32,8 @@
 // appleseed.foundation headers.
 #include "foundation/core/concepts/noncopyable.h"
 
+#include "mainwindow/mainwindow.h"
```

Comment:
Yes, it is ok. It's more work, but it's better and we already do it everywhere in the rest of the code.